EN/Moodles_EN.txt

```diff
@@ -54,14 +54,14 @@ Moodles_EN = {
 	Moodles_angry_desc_lvl4 = "90%^ Angry<br>no effects",
 	-- Stress
 	Moodles_stress_desc_lvl1 = "10%^ Stressed<br>foraging vision reduced",
-	Moodles_stress_desc_lvl2 = "25%^ Stressed<br>less foraging vision<br>-10% melee damage<br>unhappiness increase",
-	Moodles_stress_desc_lvl3 = "50%^ Stressed<br>even less foraging vision<br>-20% melee damage<br>unhappiness increases fast",
-	Moodles_stress_desc_lvl4 = "75%^ Stressed<br>worse foraging vision<br>-30% melee damage<br>unhappiness quickly increases",
+	Moodles_stress_desc_lvl2 = "25%^ Stressed<br>less foraging vision<br>-0.1 damage<br>unhappiness increase",
+	Moodles_stress_desc_lvl3 = "50%^ Stressed<br>even less foraging vision<br>-0.2 damage<br>unhappiness increases fast",
+	Moodles_stress_desc_lvl4 = "75%^ Stressed<br>worse foraging vision<br>-0.3 damage<br>unhappiness quickly increases",
 	-- Panic
 	Moodles_panic_desc_lvl1 = "6%^ Panicked<br>foraging vision reduced<br>-1,3% crit. chance<br>worse reload time<br>worse reload speed<br>worse rack time",
-	Moodles_panic_desc_lvl2 = "30%^ Panicked<br>less foraging vision<br>-2,3% crit. chance<br>-10% melee damage<br>gets up from sitting faster<br>worse knockback<br>worse knockdown chance<br>worse accuracy<br>worse reload time and speed<br>worse rack time",
-	Moodles_panic_desc_lvl3 = "65%^ Panicked<br>even less foraging vision<br>-3,9% crit. chance<br>-20% melee damage<br>gets up from sitting faster<br>worse knockback<br>worse knockdown chance<br>worse accuracy<br>worse reload time and speed<br>worse rack time",
-	Moodles_panic_desc_lvl4 = "80%^ Panicked<br>worse foraging vision<br>-5,2% crit. chance<br>-30% melee damage<br>gets up from sitting faster<br>vision narrows<br>worse knockback<br>worse knockdown chance<br>worse accuracy<br>worse reload time and speed<br>worse rack time",
+	Moodles_panic_desc_lvl2 = "30%^ Panicked<br>less foraging vision<br>-2,6% crit. chance<br>-0.1 melee damage<br>gets up from sitting faster<br>worse knockback<br>worse knockdown chance<br>worse accuracy<br>worse reload time and speed<br>worse rack time",
+	Moodles_panic_desc_lvl3 = "65%^ Panicked<br>even less foraging vision<br>-3,9% crit. chance<br>damage -0.2 melee, -0.4 distance (<6 aiming)<br>gets up from sitting faster<br>worse knockback<br>worse knockdown chance<br>worse accuracy<br>worse reload time and speed<br>worse rack time",
```

Comment:
Could you elaborate on what "<6 aiming" means?

```suggestion
	Moodles_panic_desc_lvl3 = "65%^ Panicked<br>even less foraging vision<br>-3,9% crit. chance<br>-%20 melee damage, -%40 view distance (<6 aiming)<br>gets up from sitting faster<br>worse knockback<br>worse knockdown chance<br>worse accuracy<br>worse reload time and speed<br>worse rack time",
```

Reply:
<6 aiming is for the perk, the  reduction only takes place if you're <6 at the aiming perk

```java
                  if (var2.isRanged() && var1.getPerkLevel(PerkFactory.Perks.Aiming) < 6 && var1.getMoodles().getMoodleLevel(MoodleType.Panic) > 2) {
                     var34 -= (float)var1.getMoodles().getMoodleLevel(MoodleType.Panic) * 0.2F;
                     MoodlesUI.getInstance().wiggle(MoodleType.Panic);
                  }
```

For the change in % I still do not agree as, I believe it to be false (answered about it before)

Reply:
That seems right.
I'll suggest rewording it then since it was a bit unclear (will post the new review soon).